lib/ansible/modules/copy.py

```diff
@@ -260,7 +260,7 @@
     description: Permissions of the target, after execution.
     returned: success
     type: str
-    sample: 0644
+    sample: '"0644"'
```

Comment:
```suggestion
    sample: '0644'
```
Double quotes would simply be wrong, since the module does not have quotes in its output.